src/components/manage/Widgets/__snapshots__/IdWidget.test.jsx.snap

```diff
@@ -188,7 +188,7 @@ exports[`IdWidget renders an id widget with an reserved name 1`] = `
 exports[`IdWidget renders an id widget with invalid characters 1`] = `
 <div>
   <div
-    class="error inline field text field-wrapper-my-field"
+    class="inline field text field-wrapper-my-field"
   >
```

Comment:
Why the test do not error now? If that was the intention, it should continue erroring :)

Reply:
In the issue, I saw this: `Allow only 7-bit bytes characters (https://zopeschema.readthedocs.io/en/latest/fields.html#ascii)`
This include uppercase letters, so I figured it must be permitted. Did I get it wrong? In case uppercase is valid now, I will change the test file to have an invalid input so it returns error.

Reply:
@stevepiercy I think uppercase letters should not be allowed in an `id`, although I think that Plone then normalizes them, I guess we should not allow them to enter, as it can lead to confusion.